data/DukeTask.txt

```diff
@@ -0,0 +1,5 @@
+
+[T][ ]          kill humans
+[T][ ]          kill pokemon
```

Comment:
Erhmmm, I hope it will never be mark done😰  Should have added this into .gitignore.